src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,9 @@
+function ArticleCard({ title }) {
+  return (
+    <>
```

Comment:
consider what other information the user would like to see regarding each article before they click on it and what would be the best way to display this information to the user? See this: https://www.refactoringui.com/previews/labels-are-a-last-resort